test/test-govet.sh

```diff
@@ -25,7 +25,7 @@ function simplify-gocase() {
 
 function token-coloncheck() {
 	# add quotes to avoid matching three X's
-	if grep -Ei "[\/]+[\/]+[ ]*+(FIXME[^:]|TODO[^:]|X"'X'"X[^:])" "$1"; then
```

Comment:
What's this about?

Reply:
As mentioned in an earlier comment: https://github.com/purpleidea/mgmt/pull/304#issuecomment-364084335

This is a fatal error for grep on OSX (not a warning as I stated above). And I know a lot of regex but don't know what a repetition operator applied to an repetition operator is suppose to do?

Reply:
Yeah, I have no idea either :/
Assuming it still works as expected, then ACK :)